docs/notebooks/geolift1.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>Maybe just a single sentence what we're seeing and what the values represent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done - in an upcoming commit.</p>